demo/python/orchestrator-cluster-remote-control/run-distributed.sh

```diff
@@ -1 +1 @@
-../../../host/dmlc-core/tracker/dmlc-submit --log-level DEBUG --cluster ssh --host-file hosts.config --num-workers $1 --worker-memory 4g python3 server/enclave-cluster-serve.py
+../../../host/dmlc-core/tracker/dmlc-submit --log-level DEBUG --cluster ssh --host-file hosts.config --num-workers $1 --worker-memory 4g python3 server/enclave_serve.py
```

Comment:
I suppose this script should also start the orchestrator service?

Reply:
I separated the two so that we could see output from the tracker and from the orchestrator separately. i.e. you can start the tracker in one window and the orchestrator in another window to see the two outputs separately